tests/forte/pipeline_test.py

```diff
@@ -160,6 +164,66 @@ def default_configs(cls):
         return config
 
 
+@ddt
+class PredictorPipelineTest(unittest.TestCase):
+    class DummyModel:
+        "Dummy Model."
+        def __call__(self, batch):
+            "Dummy model does nothing."
+            pass
+
+    class DummyPredcictor(Predictor):
+        "Dummy Predicotr."
+        def predict(self, batch):
+            return {}
+
+    @data(2, 4, 8)
```

Comment:
Do you mind make more copies from the pipeline tests? There we try many different setups of the batch_size.

https://github.com/asyml/forte/blob/master/tests/forte/pipeline_test.py#L258